### PR TITLE
feat(text-input): initialize Svelte

### PR DESCRIPTION
**Pull request by thollander**
## Description

Initialize the svelte version for text-input.
Needed to add Roboto-Condensed on CSS package of text-input. 

See #5

### CODE REVIEW

File: packages/sources/css/src/components/text-input/src/index.css [hunk 1]
```diff
@@ -3,6 +3,12 @@
 @import '@vtmn/css-design-tokens/src/spacings';
 @import '@vtmn/css-design-tokens/src/typography';
 
+.vtmn-text-input_label,
+.vtmn-text-input,
+.vtmn-text-input_helper-text {
+  font-family: 'Roboto Condensed', 'system-ui', '-apple-system', sans-serif;
```
Comment (lauthieb) [inline packages/sources/css/src/components/text-input/src/index.css:9]:
```suggestion
  font-family: var(--vtmn-typo_font-family);
```

Comment (lauthieb) [reply packages/sources/css/src/components/text-input/src/index.css:9]: Added the CSS Custom Properties provided by our design tokens package to add Roboto.
For your information, Roboto Condensed is not the font that we have chosen for the text input component (due to lisibility).
Another information, following our survey #4, the font-family of the button will also change in the future from Roboto Condensed to Roboto.

File: packages/sources/svelte/src/components/VtmnTextInput.svelte [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   * Whether success should be shown or not
+   * @type {boolean}
+   */
+  export let showValid = true;
+
+  /**
+   * Whether error should be shown or not
+   * @type {boolean}
+   */
+  export let showError = true;
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/VtmnTextInput.svelte:44]: Why do you propose props for "show"?
As you can see for example with material-ui: https://material-ui.com/components/text-fields/
They only propose one prop "error", if the consumer tells that the error is true, so they directly show the error style.
I don't really understand why there is two props :)

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thank you Terence!
Just few suggestion & question.

File: packages/sources/svelte/src/components/VtmnTextInput.svelte (hunk 2 above)
Comment (thollander) [reply packages/sources/svelte/src/components/VtmnTextInput.svelte]: The use case would be : I want to have an error management but without having the "green/red" information for the user. 
But that's fine, I removed it, let's keep it simple for now :)

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
Thanks a lot @thollander :tada:

**Pull request merged**